advanced/Scripts/chronometer.sh

```diff
@@ -64,9 +64,9 @@ printFunc() {
     # If there is additional text, define max length of text_main
     if [[ -n "$text_addn" ]]; then
         case "$scr_cols" in
-            [0-9]|1[0-9]|2[0-9]|3[0-9]|4[0-4]) text_main_max_len="9";;
-            4[5-9]) text_main_max_len="14";;
-            *) text_main_max_len="19";;
+            [0-9]|1[0-9]|2[0-9]|3[0-9]|4[0-4]) text_main_max_len="9" ;;
+            4[5-9]) text_main_max_len="14" ;;
+            *) text_main_max_len="19" ;;
```

Comment:
It's good to apply an already chosen coding style consistently, but here you change one: This space was never present in any case. If you use a tool for this, it should be possible to remove this rule, respectively invert it to have this space omitted consistently.

The same may be done for the array key indentation.

A consistent condition and loop indentation is IMHO the most important thing. I remember that I needed a second and a third look in the installer by times to find the end of a condition statement or to not accidentally take a `fi` for the parent `if` statement closing due to false indentation while it was actually for a child statement, or the other way round. Trailing spaces come next, mostly because they leave ugly green highlight in common editors and, well, are wasted bytes. So if there is no more `if condition; then`/`for statement; do` alignment (having these consistent in a single line) required, then probably it's best to only apply indentation and trailing space fixes for now, as the benefits are understandable, while everything else is pretty much a question of personal taste only.

Reply:
Unfortunately, `beautysh` does not support customization beyond indentation level. It is not the most powerful formatter, and I have already done a couple of manual fix-ups, where it was being over-aggressive.
I am happy to make another pass and remove this particular formatting change, and if you can explain what you mean about 'array key format' then I'll change that too.

Reply:
With the array I mean:
- https://github.com/pi-hole/pi-hole/pull/4085/files#diff-8e13fa9927aabae5439f4ae9d8736f56ab9220296dd931a69cab1e97830a63e3L162
- https://github.com/pi-hole/pi-hole/pull/4085/files#diff-595630a29a855f6d667a84ca0662042e826bf3ec56322ef61d4a6ef149147d23R1093

I don't see any clear preference for this, so I wouldn't touch it. My personal preference (which does not count) would be:
```sh
array=(
    item1
    item2
    ...
)
```
But `beautysh` makes this of it:
```sh
array=(item1
    item2
    ...)
```

And this one also looks strange:
- https://github.com/pi-hole/pi-hole/pull/4085/files#diff-595630a29a855f6d667a84ca0662042e826bf3ec56322ef61d4a6ef149147d23R1954

Reply:
Okay, I have further reduced the changes, they should now only contain start of line whitespace adjustments, except where such adjustments would affect strings, bring comments out of line, or otherwise create confusing indentation.